fastlane/lib/fastlane/actions/swiftlint.rb

```diff
@@ -19,6 +19,7 @@ def self.run(params)
         command << " --reporter #{params[:reporter]}" if params[:reporter]
         command << supported_option_switch(params, :quiet, "0.9.0", true)
         command << supported_option_switch(params, :format, "0.11.0", true) if params[:mode] == :autocorrect
+        command << " --no-cache" if params[:no_cache] and (params[:mode] == :autocorrect or params[:mode] == :lint)
```

Comment:
Will fix these two rubocop problems:

```
[12:53:22]: ▸ fastlane/lib/fastlane/actions/swiftlint.rb:22:55: C: Style/AndOr: Use && instead of and.
[12:53:22]: ▸ command << " --no-cache" if params[:no_cache] and (params[:mode] == :autocorrect or params[:mode] == :lint)
[12:53:22]: ▸ ^^^
[12:53:22]: ▸ fastlane/lib/fastlane/actions/swiftlint.rb:22:90: C: Style/AndOr: Use || instead of or.
[12:53:22]: ▸ command << " --no-cache" if params[:no_cache] and (params[:mode] == :autocorrect or params[:mode] == :lint)
[12:53:22]: ▸ ^^
```

```suggestion
        command << " --no-cache" if params[:no_cache] && (params[:mode] == :autocorrect || params[:mode] == :lint)
```

Reply:
Feedback addressed @janpio